assignment4.txt

```diff
@@ -0,0 +1,27 @@
+/*Rebecca Hale Assignment 4 Part 2*/
+1. Strings can be concatenated in C by using the function
+strcat(string1,string2).  This function automatically concatenates 
+the strings and puts them into string1.
+2. If a static array is not initialized, then the array is not given an
+automatic size.  In fact, if the function sizeof() is used to locate the
+size of the array without first initializing the array, an error will
+occur.  
+3. A multidimensional array of size 64 can be declared using any dimensions
+that when multiplied together equal 64.  For instance, int x[8][8] or 
+int x[2][32] or int x[4][16]. 
+4.
+a. strcmp() is a function that compares the lexicographical order of two
```

Comment:
nice! you should have explained that a returned 0 means that the two strings are equal though